.vscode/settings.json

```diff
@@ -126,9 +126,6 @@
         },
         "search.showLineNumbers": true,
         "telemetry.enableTelemetry": false,
-        "terminal.integrated.copyOnSelection": true,
-        "terminal.integrated.rightClickBehavior": "paste",
-        "terminal.integrated.scrollback": 5000,
```

Comment:
Can you leave the extended scrollback (assuming this is larger than the current default)?

Reply:
I can, but it still messes up my local config, which is 10000. This seems like a personal preference setting to me.

Reply:
FWIW, the VS Code default is 1000.

Reply:
Bump it to 10000 then? I'm trying to be pragmatic and get as close as we can to a good experience out of the box with minimal effort.

Reply:
I'll just leave if it is really that important.